blueprint/abac/platform.md

```diff
@@ -542,7 +542,14 @@ And OS In <Not configured>
 
 #### APIs
 
-* SIEM: `Not configured`
+* SIEM
```

Comment:
Are these the same settings for all SIEMs or just Microsoft's? Understand this came from the hybrid doc.

Reply:
The app registration will be the same for all SIEMs - essentially when you create the app registration to will give it to you in a downloadable format to provide your siem. The actual configuration on the SIEM side will change based on the SIEM in use.

![image](https://user-images.githubusercontent.com/81546609/119299853-13094d80-bca3-11eb-8ca4-61252bcb08ea.png)